compiler/rustc_driver_impl/src/lib.rs

```diff
@@ -715,13 +748,13 @@ fn print_crate_info(
 
                 cfgs.sort();
                 for cfg in cfgs {
-                    safe_println!("{cfg}");
+                    println_info!("{cfg}");
                 }
             }
             CallingConventions => {
                 let mut calling_conventions = rustc_target::spec::abi::all_names();
                 calling_conventions.sort_unstable();
-                safe_println!("{}", calling_conventions.join("\n"));
+                println_info!("{}", calling_conventions.join("\n"));
             }
             RelocationModels
             | CodeModels
```

Comment:
it looks like `codegen_backend.print` prints to stdout. Apparently only LLVM has this printing logic, but I think it should be made consistent.

Reply:
I updated the printing in the LLVM backend.